akka-cluster-sharding/build.sbt

```diff
@@ -0,0 +1,18 @@
+import akka.{ AkkaBuild, Dependencies, Formatting, MultiNode, Unidoc, OSGi }
+import com.typesafe.tools.mima.plugin.MimaKeys
+
+AkkaBuild.defaultSettings
+
+Formatting.formatSettings
+
+Unidoc.scaladocSettingsNoVerificationOfDiagrams
+
+Unidoc.javadocSettings
+
+OSGi.clusterSharding
+
+MultiNode.multiJvmSettings
+
+libraryDependencies ++= Dependencies.clusterSharding
```

Comment:
for these we might switch to declaring some libraries centrally and then using symbolic names in the sub-projects—this would keep the version information in one place while also keeping the dependency management local